airflow/models/dag.py

```diff
@@ -3378,6 +3394,8 @@ class DagModel(Base):
     processor_subdir = Column(String(2000), nullable=True)
     # String representing the owners
     owners = Column(String(2000))
+    # Display name of the dag
+    _dag_display_name = Column("dag_display_name", String(2000), default="")
```

Comment:
Same as above

Reply:
Same as above: The model is defined as far as I understand the code to use the `task_id` if no display name is given. This is good as it saved space in the DB (if no display name given) as well as allows migration (else during migration the column needs to be filled).
I'd propose to keep it this way.